Formula/ssh-agent-filter.rb

```diff
@@ -19,7 +19,22 @@ def install
   end
 
   test do
-    system "#{bin}/ssh-agent-filter", "-V"
+    (testpath/"test.sh").write <<-EOS.undent
+      mydir=$(pwd)
+      cd $(mktemp -d)
+      ssh-keygen -q -N '' -f keyA
```

Comment:
This needs considerable processing time for finding primes. You might use `-t ed25519` to select a key type that is much faster to generate. In that case you should also use `-C keyA` to set the comment or ssh-add/ssh-agent might use the embedded comment instead of the filename.